src/System.Collections.NonGeneric/tests/Performance/Perf.ArrayList.cs

```diff
@@ -0,0 +1,476 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Xunit.Performance;
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.Collections.Tests
+{
+    public class Perf_ArrayList
+    {
+        private static volatile int _temp;
+
+        private ArrayList CreateArrayListOfInts(int size = 100000)
+        {
+            var arrayListOfInts = new ArrayList(size);
+            for (int element = 0; element < size; element++)
+            {
+                arrayListOfInts.Add(element);
+            }
+
+            return arrayListOfInts;
+        }
+
+        [Benchmark(InnerIterationCount = 2000000)]
+        public void Add()
+        {
+            int innerIteractionCount = (int)Benchmark.InnerIterationCount;
```

Comment:
nit, some places 'Iteraction' instead of "Interaction"

Reply:
I actually wanted to say "Iteration". :))